app/services/claims/context_mapper.rb

```diff
@@ -22,9 +22,25 @@ def available_schemes
       end
     end
 
+    def available_compreensive_schemes
```

Comment:
🗒  As I've mentioned before, I'm not particularly attached to these name, so happy to change this to something that makes more sense to everyone.